tests/data_source_test.go

```diff
@@ -91,18 +91,18 @@ func TestDataSource(t *testing.T) {
 	err = ctl.removeLicense()
 	a.NoError(err)
 	err = ctl.patchDataSource(adminDataSource.DatabaseID, api.DataSourcePatch{
-		ID:           readOnlyDataSource.ID,
-		HostOverride: &dataSourceNewHostOverride,
-		PortOverride: &dataSourceNewPortOverride,
+		ID:   readOnlyDataSource.ID,
```

Comment:
Need to update the test to reflect the new behavior

Reply:
This probably doesn't need to change for now. Will rebuild the test for instance datasource updates.